examples/lib/p5.js

```diff
@@ -13454,6 +13454,32 @@ p5.Renderer2D.prototype.stroke = function() {
   ctx.strokeStyle = color.toString();
 };
 
+p5.Renderer2D.prototype._tintFillStyle = function() {
+  var hue = this._pInst.hue(this._tint);
+
+  if (hue !== 0) {
+    return 'hsl(' + hue + ', 100%, 50%)';
+  }
+
+  // Calculate saturation and lightness for hue with value 0.
+  // Reference for solution: https://gist.github.com/mjackson/5311256
+  var r = this._tint[0] / 255;
+  var g = this._tint[1] / 255;
+  var b = this._tint[2] / 255;
+  var max = Math.max(r, g, b);
+  var min = Math.min(r, g, b);
+  var saturation, lightness = (max + min) / 2;
+
+  if (max == min) {
+    saturation = 0; // achromatic
+  } else {
+    var delta = max - min;
+    saturation = lightness > 0.5 ? delta / (2 - max - min) : delta / (max + min);
+  }
+
+  return 'hsl(' + hue + ', ' + (saturation*100) + '%, ' + (lightness*100) + '%)';
```

Comment:
Can we use `this._pInst.brightness(this._tint);` and `this._pInst.saturation(this._tint);` to calculate these values for us?

Reply:
Re: your comment in the PR description, these should work for any p5.Color instance.  They're stored as 4 normalized values internally.

Reply:
I tried using those, but from my understanding through testing and the docs, those methods only work if `colorMode` is HSB or HSL. I messed around with changing the color mode and that didn't help, but I also could've been going about it incorrectly